taxi_service/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for taxi_service project.
+
+Generated by 'django-admin startproject' using Django 4.2.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-lou)kt@!p9#i_9r(p36u*2v)b6%7g=9i3ee_nt(id%h@wpll%^"
```

Comment:
Remove your secret key and do not store it in a repo, it is sensitive information.